src/containers/PipelineRun/PipelineRun.js

```diff
@@ -260,11 +260,20 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
     const logContainer = (
       <Log
-        key={stepName}
-        stepName={stepName}
-        podName={taskRun.pod}
+        fetchLogs={async () => {
```

Comment:
can we extract this for re-use (PipelineRun, TaskRun, TaskRuns)? it would take the run and stepStatus as params